site/pages/gcweb-theme/static-header-footer/bootstrap-3.html

```diff
@@ -16,24 +16,33 @@
 
 <header>
 	<div id="wb-bnr" class="container">
-		<section id="wb-lng" class="text-right">
-			<h2 class="wb-inv">Language selection</h2>
-			<ul class="list-inline margin-bottom-none">
-				<li><a lang="fr" href="archived-fr.html">Français</a></li>
-			</ul>
-		</section>
 		<div class="row">
-			<div class="brand col-xs-5 col-md-4" property="publisher" typeof="GovernmentOrganization"> <a href="https://www.canada.ca/en.html" property="url"><img src="https://wet-boew.github.io/themes-dist/GCWeb/assets/sig-blk-en.svg" alt="" property="logo"><span class="wb-inv" property="name"> Government of Canada / <span lang="fr">Gouvernement du Canada</span></span></a>
+			<section id="wb-lng" class="col-sm-12 col-xs-3 text-right">
+				<h2 class="wb-inv">Language selection</h2>
+				<ul class="list-inline margin-bottom-0">
+					<li>
+						<a lang="fr" hreflang="fr" href="home-fr.html">
+							<span class="hidden-xs">Français</span>
+							<abbr title="Français" class="visible-xs h3 mrgn-tp-sm mrgn-bttm-0">FR</abbr>
+						</a>
+					</li>
+				</ul>
+			</section>
+			<div class="brand col-xs-9 col-sm-5 col-md-4" property="publisher" typeof="GovernmentOrganization">
+				<a href="https://www.canada.ca/en.html" property="url">
+					<img src="https://wet-boew.github.io/themes-dist/GCWeb/assets/sig-blk-en.svg" alt="" property="logo"><span class="wb-inv" property="name"> Government of Canada / <span lang="fr">Gouvernement du Canada</span></span>
+				</a>
 				<meta property="areaServed" typeof="Country" content="Canada">
 				<link property="logo" href="https://wet-boew.github.io/themes-dist/GCWeb/assets/wmms-blk.svg">
 			</div>
-			<section id="wb-srch" class="col-lg-8 text-right">
+			<section id="wb-srch" class="col-xs-12 col-sm-5 col-sm-offset-2 col-md-4 col-md-offset-4 col-lg-offset-4">
 				<h2>Search</h2>
 				<form action="#" method="post" name="cse-search-box" role="search" class="form-inline">
 					<div class="form-group">
 						<label for="wb-srch-q" class="wb-inv">Search Canada.ca</label>
 						<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Search Canada.ca">
-						<datalist id="wb-srch-q-ac"> </datalist>
+						<datalist id="wb-srch-q-ac">
+						</datalist>
 					</div>
 					<div class="form-group submit">
```

Comment:
```suggestion
					<div class="form-group wb-srch-sub">
```
(See comment in the CSS.)